.github/workflows/test.yml

```diff
@@ -0,0 +1,102 @@
+---
+# Source: https://github.com/hendrikmaus/rust-workflows
+#
+# Continuous Integration Workflow For Rust
+name: CI
+
+# Define the triggers; usually merges to the repository
+# default branch and pull-requests
+on:
+  push:
+    branches:
+      - main
+      - master
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    paths-ignore:
+      - "docs/**"
+      - "**.md"
+  # The following makes this workflow re-usable by other repositories
+  # without having to copy the entire thing.
+  #
+  # See https://docs.github.com/en/actions/learn-github-actions/reusing-workflows
+  workflow_call: {}
+
+env:
+  CARGO_TERM_COLOR: always
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-20.04
```

Comment:
What's the reasoning for using `ubuntu-20.04` here? `ubuntu-latest` is used below.

cc @hendrikmaus